dev-lang/php/php-7.4.16.recipe

```diff
@@ -34,6 +40,7 @@ PROVIDES="
 	cmd:php_cgi
 	cmd:php_config
 	cmd:phpize
+	cmd:php_fpm
```

Comment:
Needs looking into, at the moment php_fpm is installed in $binDir instead of $commandBinDir